lecture_2/homework/app/controllers/clans/warriors_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Clans
+  
+  class WarriorsController < ApplicationController
+
+    def show
+      render json: serializer.new(warrior)
```

Comment:
A próbowałeś zrobić ten `include` klanu i broni tutaj? Bo tak jak jest teraz to wiemy, że wojownik ma broń i jest w klanie, ale żeby poznać szczegóły będziemy potrzebować dodatkowych zapytań.

Reply:
Tak to prawda muszę to dopisać, już chciałem tą logike rozwinąc przy refactorze odrazu.